requirements.txt

```diff
@@ -1,4 +1,4 @@
--e git+https://github.com/tiptapinc/zerog.git@0.0.37#egg=zerog
+-e git+https://github.com/tiptapinc/zerog.git@SV2-2393-add-dump-handler#egg=zerog
```

Comment:
I made comments in the adwords-service pull request about how I would handle the versioning a little differently